### PR TITLE
Experimental design content updates

### PR DESCRIPTION
**Pull request by briandeconinck**
Content updates to [align content with the new experimental design flow in Mural](https://github.com/department-of-veterans-affairs/va.gov-team/issues/96032).

### CODE REVIEW

File: src/_about/contributing/suggest-an-addition-or-update.html [hunk 1]
```diff
@@ -42,60 +45,59 @@ <h3 id="fill-out-a-request">Fill out a request</h3>
...
     <p>
-      Note that you do not need to have anything coded at this point, and in fact, we suggest not having anything coded since ideally ideas would be surfaced to the council before teams put effort into building new components.  
+      You must validate the design of your new component or pattern with user research. <a href="https://depo-platform-documentation.scrollhelp.site/research-design/research-at-va">Learn about conducting research at VA.</a> You may conduct research before presenting your work to the Design System Council, or you may present to the Design System Council and receive feedback before you complete a research study. 
     </p>
     <p>
-      Additionally, the artifacts you include do not need to be high fidelity. They only need to be able to illustrate your concept. We do, however, encourage some thoughts on how your component or pattern works on mobile.
+      Note that even if the Design System Council approves your component or pattern before you conduct research, you must still validate your work with user research and present your findings to the Design System Council before it can be included in the design system.
```
Comment (humancompanion-usds) [inline src/_about/contributing/suggest-an-addition-or-update.html:60]: This is not my expectation. They can come back with their findings but they can also just attached them to the issue and ping the DST.

File: src/_about/contributing/suggest-an-addition-or-update.html [hunk 2]
```diff
@@ -42,60 +45,59 @@ <h3 id="fill-out-a-request">Fill out a request</h3>
...
+      You will present your work to the Design System Council at an upcoming meeting (unless you have requested an <a href="#async-review">asynchronous review</a>). The Design System Council meets every other Thursday at 11am Eastern. Experimental design requests will always be prioritized in this meeting, and teams can assume generally that they will be able to present the same week of the request.
+    </p>
+    <p>
+      When you are ready to present, post in the <a href="https://dsva.slack.com/archives/C01DBGX4P45">#platform-design-system</a> Slack channel and tag Carol Wong to alert the team that you have filled out an issue and would like to get on the council's agenda. During the meeting, the Design System Council will evaluate the request and make a decision.
```
Comment (humancompanion-usds) [inline src/_about/contributing/suggest-an-addition-or-update.html:71]: Can we use site.slack_channel_link and {{ site.slack_channel_name }} like we did before?

**Review by humancompanion-usds — CHANGES_REQUESTED** [review]
Just left 2 comments.

**Review by shiragoodman — APPROVED** [review]

File: src/_about/contributing/suggest-an-addition-or-update.html (hunk 2 above)
Comment (briandeconinck) [reply src/_about/contributing/suggest-an-addition-or-update.html]: Done.

File: src/_about/contributing/suggest-an-addition-or-update.html (hunk 1 above)
Comment (briandeconinck) [reply src/_about/contributing/suggest-an-addition-or-update.html]: Rewrote to match these expectations.

**Review by it-harrison — APPROVED** [review]

**Review by humancompanion-usds — APPROVED** [review]

**Pull request merged**